.travis.yml

```diff
@@ -105,37 +55,32 @@ matrix:
         - alpine ../scripts/check-static-build.sh || travis_terminate 1
         - alpine ./tst/webrtc_client_test
       after_failure: skip # no coredumps in container
-
-    # Cross-compilation to ARM, no tests are run
-    - name: "ARM Cross-compilation"
-      os: linux
-      addons:
-        apt:
-          packages:
-            - gcc-arm-linux-gnueabi
-            - g++-arm-linux-gnueabi
-            - binutils-arm-linux-gnueabi
-      compiler: gcc
-      before_install:
-        # TODO: Remove the following line. This is only a workaround for enabling IPv6, https://github.com/travis-ci/travis-ci/issues/8891.
-        - sudo sh -c 'echo 0 > /proc/sys/net/ipv6/conf/all/disable_ipv6'
-      before_script:
-        - export CC=arm-linux-gnueabi-gcc CXX=arm-linux-gnueabi-g++
-        - mkdir build && cd build
-        - cmake .. -DBUILD_OPENSSL=TRUE -DBUILD_OPENSSL_PLATFORM=linux-generic32 -DBUILD_LIBSRTP_HOST_PLATFORM=x86_64-unknown-linux-gnu -DBUILD_LIBSRTP_DESTINATION_PLATFORM=arm-unknown-linux-uclibcgnueabi
-      script: make
-
-    - name: "mbedTLS - Linux GCC 4.4 Build"
-      os: linux
-      compiler: gcc
-      before_install:
-        # TODO: Remove the following line. This is only a workaround for enabling IPv6, https://github.com/travis-ci/travis-ci/issues/8891.
-        - sudo sh -c 'echo 0 > /proc/sys/net/ipv6/conf/all/disable_ipv6'
-        - sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
-        - sudo apt-get -q update
-        - sudo apt-get -y install gcc-4.4
-        - sudo apt-get -y install gdb
-      before_script: mkdir build && cd build && cmake .. -DBUILD_TEST=TRUE -DUSE_OPENSSL=OFF -DUSE_MBEDTLS=ON
+    
```

Comment:
Although msan and windows runs are not passing in Travis as well, let us not comment these out.

Reply:
Uncommented it in the latest commit!